website/docs/r/servicebus_namespace_customer_managed_key.html.markdown

```diff
@@ -0,0 +1,161 @@
+---
+subcategory: "Messaging"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_servicebus_namespace"
+description: |-
+  Manages a ServiceBus Namespace Customer Managed Key.
```

Comment:
I realise ServiceBus is written together as one word in a lot of our documentation, but I think this should actually be split into two words. Could you split it in this doc and we can update the others at a later point?